wire-library/wire-swift-generator/src/main/java/com/squareup/wire/swift/SwiftGenerator.kt

```diff
@@ -585,6 +584,36 @@ class SwiftGenerator private constructor(
       PROTO_3 -> proto3Codable
     }
 
+  private fun heapCodableExtension(
```

Comment:
Good catch on this. Did we not have a test for heap type encode/decode?

Reply:
Nope... there's only a _single_ heap type right now and it would not be feasible to write a test for it.

Reply:
New test: https://github.com/square/wire/pull/2379/commits/110661eb7d992757ca00a91306f500cd0e6db18b